showast/rendering/graphviz.py

```diff
@@ -15,11 +16,13 @@
 from showast.asts import recurse_through_ast
 
 _LOCATION_ATTRS = [('lineno', 'col_offset'), ('end_lineno', 'end_col_offset')]
-
+_NEWLINE_SOURCE_CHAR = '\\n'
 
 def _bold(label):
     return '<<B>{}</B>>'.format(label)
 
+def _unparse(node):
```

Comment:
Changes introduced do not guard against failure. Currently, if `ast.unparse()` does not return a string, or the resulting node label is unintelligible to **GraphViz** displaying may fail and the only work around is to disable showing location info and or unparsed.